website/content/about/SustainabilityUseCasesAndLandscape2023.md

```diff
@@ -4,7 +4,7 @@ description: This captures the known and ongoing sustainability efforts within t
 slug: landscape
 ---
 
-*This document is in version 0.1 and contains gaps which will be addressed in the upcoming versions. Contributions are very welcome!*
+*This document is in version 23.06 and contains gaps which will be addressed in the upcoming versions. Contributions are very welcome!*
```

Comment:
```suggestion
*This document was published on 23.06. We are aware that this document contains gaps that will be addressed in future releases. Contributions are very welcome!*
```

When we mention "version" at the beginning, "This document is in version 23.06", you might think that it is the major version 23 and minor version 6.

Reply:
@catblade `We are aware that this document contains gaps that will be addressed in future releases`WDYT about, we add placeholders to the document? Maybe it would disturb the reading flow.... But it could help the contributors to complement the document.